demos/remote_attestation/dcap/c_app/dcap_c_test.c

```diff
@@ -69,7 +69,11 @@ void main() {
     memset(p_quote_buffer, 0, quote_size);
 
     sgx_report_data_t report_data = { 0 };
-    char *data = "ioctl DCAP report data example";
+    char *data = "ppml";
+    if (args[1] != NULL && args[1] != "") {
+    	data = args[1];
```

Comment:
Using unchecked data as report data is a security bug. Because a hacker could use this interface to generate any reports.  It is not a good demo for users.